Primo.js

```diff
@@ -761,4 +785,4 @@ var testCases = [
 		]
 	}
 ]
-/** END TEST CASES **/
+/** END TEST CASES **/
```

Comment:
Add another test case for http://digitale.beic.it/primo_library/libweb/action/display.do?doc=39bei_digitool2018516 ?